libc/src/math/amdgpu/llround.cpp

```diff
@@ -1,18 +0,0 @@
-//===-- Implementation of the GPU llround function ------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/math/llround.h"
-#include "src/__support/common.h"
-
-namespace LIBC_NAMESPACE {
-
-LLVM_LIBC_FUNCTION(long long, llround, (double x)) {
-  return __builtin_llround(x);
```

Comment:
We should just support these. I don't really get why these exist, they're implemented in the HIP headers as the cast of the regular round

Reply:
@lntue is this strictly true? `round` returns a double, which I'm unsure if that correctly converts to a `long long int` when casted. I would guess not because there would be gaps at the extreme ends.

Reply:
Can't seem to find a value that doesn't return the same when casted, so maybe you can just cast it.

Reply:
Changed it to just use a cast on the builtin.